server/package.json

```diff
@@ -4,12 +4,18 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest",
```

Comment:
@pcgamer1 I see no script to run the server? how do you run it locally?

Reply:
Can we have a npm run dev which points to node index.js to run locally?

Reply:
This could've been a separate commit, the value that it gives is different. Why have you forced fitted it into the tests changes commit. 
1 commit = 1 piece of work not coupling unrelated things together. 
Makes it very difficult to read